### ISSUE TITLE
Cannot call "this.props.toRoute" in titleComponent

### ISSUE THREAD

**Issue by anhvn** [react-native-simple-router-community/react-native-simple-router#49]
# Hi,

I'm writing an application that need to navigate to other page from titleComponent in the router (actually it is a searchBar).  But I cannot call "this.props.toRoute" in this component. This function is called successfully in leftCorner but is undefined in titleComponent.

So what can I do to call toRoute function in titleComponent?

The code of searchBar

```
 var SearchBar = React.createClass({
     searchLetters: function (text) {
         console.log(text);
         this.props.toRoute({
             name: "Result for: " + text,
             component: SearchPage,
             passProps: { strSearch: text }
         });
     },
     render() {
         return (
             <TextInput
                 style={styles.input} placeholder="Search letters.."
                 placeholderTextColor="rgba(255, 255, 255, 0.5)"
                 returnKeyType="search"
                 enablesReturnKeyAutomatically={true}
                 onEndEditing={ event => this.searchLetters(event.nativeEvent.text) }
             />
         )
     }
 });
```

The index code

```
 var firstRoute = {
   name: 'Letters',
   component: SearchPage,
   titleComponent: SearchBar,
   passProps: { strSearch: '' }
 };

 var Letters = React.createClass({
   render() {
     return (
         <Router firstRoute={firstRoute} headerStyle={ styles.routerHeader } />
     );
   }
 });
```

Thank you!

**Comment by charpeni** [comment]
Hi @anhvn.

`toRoute` isn't available from titleComponent at this time, it will be in the next release.

Until then, you can use `titleProps` to pass the `toRoute` function. Unfortunalty, it will not work for first route.

``` javascript
this.props.toRoute({
  name: 'Letters',
  component: SearchPage,
  titleComponent: SearchBar,
  titleProps: {
    toRoute: this.props.toRoute,
  },
  passProps: { strSearch: '' },
});
```

**Comment by davidLeonardi** [comment]
Closed due to landing of #68
